crux-mir/src/Mir/Language.hs

```diff
@@ -476,6 +476,25 @@ runTestsWithExtraOverrides initS bindExtra (cruxOpts, mirOpts) = do
       else
         return ExitSuccess
 
+-- | Add a prefix to the string, that will be different depending on the
+-- cases of the letters.  We use it to avoid name collision on case
+-- insensitive file systems.
```

Comment:
Can you give some concrete examples of input/output pairs to make it more obvious how this function works?